dsgrid/cli/registry.py

```diff
@@ -149,8 +161,13 @@ def list_dimensions(registry_manager, filter):
 )
 @click.pass_obj
 @click.pass_context
-def register_dimensions(ctx, registry_manager, dimension_config_file, log_message):
-    """Register new dimensions with the dsgrid repository."""
+def register_dimensions(
+    ctx, registry_manager: RegistryManager, dimension_config_file: Path, log_message: str
+):
+    """Register new dimensions with the dsgrid repository. The JSON/JSON5 filename must
+    match the data model defined by this documentation:
```

Comment:
The filename must match the data model, or the file contents must match the data model?

Reply:
I'll change `filename` to `file.`